main.tex

```diff
@@ -323,7 +323,7 @@ \section{Example}
     from astropy import coordinates, units as u, wcs
     from astroquery.skyview import SkyView
     from astroquery.vizier import Vizier
-    import pylab as pl
+    from matplotlib import pyplot as pl
```

Comment:
:+1: to this

Reply:
Super nitpicking, but according to astropy coding guidelines, this should be `import matplotlib.pyplot as plt`